### PR TITLE
Update other_limits.md

### PR DESCRIPTION
**Pull request by yifuzhang95**

### CODE REVIEW

File: pages/trading/other_limits.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+| >= $20 and < $100         | 4                               |
+| >= $100 and < $1,000       | 8                               |
+| >= $1,000 and < $10,000      | 10                              |
+| >= $10,000 and < $100,000     | 100                             |
+| >= $100,000      | 200                             |
 
 For example up to 10 open bids across all markets for a subaccount with a net collateral of $2,000.
 
-Note that limit `Immediate-or-Cancel`, `Fill-or-Kill`, and market orders on the frontend do not have this limitation.
+Note that short term orders via API And all limit `Immediate-or-Cancel`, `Fill-or-Kill`, and market orders on the frontend do not have this limitation.
```
Comment (moo-onthelawn) [inline pages/trading/other_limits.md:16]: suggestion: 

Note: This limitation does not apply to:
- API short term orders
- Frontend market orders 
- Frontend limit `Immediate-or-Cancel` or `Fill-or-Kill` orders

File: pages/trading/other_limits.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
+| Net Collateral | Long-term or Conditional orders |
+| -------------- | ------------------------------- |
+| < $20          | 0                               |
+| >= $20 and < $100         | 4                               |
+| >= $100 and < $1,000       | 8                               |
+| >= $1,000 and < $10,000      | 10                              |
+| >= $10,000 and < $100,000     | 100                             |
+| >= $100,000      | 200                             |
 
 For example up to 10 open bids across all markets for a subaccount with a net collateral of $2,000.
```
Comment (moo-onthelawn) [inline pages/trading/other_limits.md:14]: This is sort of hard to read. Suggestion:

 For example, a subaccount with a net collateral of $2000 is allowed 10 open bids across all markets.

File: pages/trading/other_limits.md [hunk 3]
```diff
@@ -2,15 +2,15 @@
 
 Subaccounts may have a limited number of open orders at any one time determined by the net collateral of the subaccount as per:
```
Comment (moo-onthelawn) [inline pages/trading/other_limits.md:3]: suggestion: The net collateral of a subaccount determines the number of open orders it is allowed. See below for limits:

**Review by moo-onthelawn — APPROVED** [review]
Would it be accurate to say that short term orders (made in any way: API / front end) aren't limited by the net collateral of a subaccount? Might be worth explicitly writing that out in the beginning rather than just removing it from the table

File: pages/trading/other_limits.md [hunk 4]
```diff
@@ -2,15 +2,19 @@
...
-| >= $10,000 and < $100,000     | 100               | 100                             |
-| >= $100,000      | 1000               | 200                             |
+| Net Collateral | Long-term or Conditional orders |
+| -------------- | ------------------------------- |
+| < $20          | 0                               |
+| >= $20 and < $100         | 4                               |
+| >= $100 and < $1,000       | 8                               |
+| >= $1,000 and < $10,000      | 10                              |
+| >= $10,000 and < $100,000     | 100                             |
+| >= $100,000      | 200                             |
```
Comment (jiajames) [inline pages/trading/other_limits.md:12]: i believe these values have changed, is it possible to instead link somewhere for the latest values? that way we don't have to update this every time

**Pull request closed without merging**